core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1681,6 +1681,29 @@ public static String translateWithCharset(String s, String transcodingName) {
     }
   }
 
+  /** Oracle's {@code CONVERT(charValue, destCharsetName[, srcCharsetName])} function,
+   * return null if s is null or empty. */
+  public static String oracleConvert(String s, String... args) {
```

Comment:
Similar to the previous suggestion, if `oracleConvert` could be changed to `convertOracle`, perhaps this would be consistent with the existing `regexpReplacePg`.